main.js

```diff
@@ -1,6 +1,69 @@
-import './style.css'
+const API_KEY = 'api_key=ba527e40ac8ca2c3848e65d14b95a2d7';
+const BASE_URL = 'https://api.themoviedb.org/3';
+const API_URL = BASE_URL + '/discover/movie?sort_by=popularity.desc&' + API_KEY;
+const IMG_URL = 'https://image.tmdb.org/t/p/w500';
+const searchURL = BASE_URL + '/search/movie?'+API_KEY;
+const movieNotFound = document.getElementById(".movie-not-found")
 
-document.querySelector('#app').innerHTML = `
-  <h1>Hello Vite!</h1>
-  <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
-`
+const main = document.getElementById('main');
+const form = document.getElementById('form');
+const search = document.getElementById('search');
+
+
+getMovies(API_URL);
+
+function getMovies(url) {
+    fetch(url).then(res => res.json()).then(data => {
+        console.log(data.results.slice(0,15))
+        showMovies(data.results.slice(0,15));
+    })
+}
+
+function showMovies(data) {
+    main.innerHTML = '';
+
+
+    data.forEach(movie => {
+        const {title, poster_path, vote_average, overview} = movie;
+        const movieEl = document.createElement('div');
+        movieEl.classList.add('movie');
+        movieEl.innerHTML = `
```

Comment:
It is better to avoid innerHTML due to performance and security issues. Please check this ![link](https://stackoverflow.com/questions/18502238/why-is-it-suggested-to-avoid-innerhtml) to know more

Reply:
> It is better to avoid innerHTML due to performance and security issues. Please check this [link](https://stackoverflow.com/questions/18502238/why-is-it-suggested-to-avoid-innerhtml) to know more

Please fix this.